include/common/logger.h

```diff
@@ -1 +1,55 @@
-class logger{};
\ No newline at end of file
+#pragma once
+#include <string>
+#include <fstream>
+#include <ctime>
+#include <chrono> 
+
+class Logger {
```

Comment:
Utility classes( like this one) we gonna name starting with lower case. "class logger"